pkg/kubelet/kubelet_pods.go

```diff
@@ -76,8 +80,75 @@ const (
 const (
 	PodInitializing   = "PodInitializing"
 	ContainerCreating = "ContainerCreating"
+
+	kubeletUser = "kubelet"
 )
 
+func parseGetSubIdsOutput(input string) (uint32, uint32, error) {
```

Comment:
Can we add some unit tests for this function?

Reply:
Also, can add a comment above to show example content we are trying to parse.

Reply:
done